CMakeLists.txt

```diff
@@ -1257,18 +1269,20 @@ write_basic_package_version_file(
   VERSION ${ZEEK_VERSION_NUMBER}
   COMPATIBILITY ExactVersion)
 
-# Install all the things.
+# Write the CMake targets file.
 export(EXPORT ZeekTargets FILE ZeekTargets.cmake NAMESPACE Zeek::)
-install(
-  FILES
-    "${CMAKE_CURRENT_BINARY_DIR}/ZeekConfig.cmake"
-    "${CMAKE_CURRENT_BINARY_DIR}/ZeekConfigVersion.cmake"
-  DESTINATION
-    "${CMAKE_INSTALL_LIBDIR}/cmake/Zeek")
-install(
-  EXPORT ZeekTargets
-  DESTINATION "${CMAKE_INSTALL_LIBDIR}/cmake/Zeek"
-  NAMESPACE Zeek::)
+
+# TODO: Check whether installing these does not break any of our packages.
```

Comment:
Can this section go away then?

Reply:
I've commented out installing the CMake package files for now, but I think it's still the way to go. I left this TODO as a reminder to do a followup to check if break any of our packages (thinking about OBS). I don't expect any issues from installing them, but I'd rather evaluate that separately from this big set of changes. 🙂